### ISSUE TITLE
Markbot issue

### ISSUE THREAD

**Issue by povoroznykmarta** [povoroznykmarta/using-markbot#1]
@hellachella
Its been a while, and Markbot still says that my website is not online.

**Comment by hellachella** [comment]
Yeah, leave it for the night. Close Markbot and open it again tomorrow. '

**Comment by hellachella** [comment]
Were you able to submit?

**Comment by povoroznykmarta** [comment]
Still no.

**Comment by hellachella** [comment]
Click on the red "Not  online" button on the sidebar and send me a screenshot,

**Comment by povoroznykmarta** [comment]
<img width="843" alt="Screenshot 2021-01-29 at 20 42 28" src="https://user-images.githubusercontent.com/77293405/106314554-91645e80-6272-11eb-94cc-3150f971cbab.png">

**Comment by hellachella** [comment]
Sign out of Markbot and back in using your github username `povoroznykmarta`. Right now you are using just your name.

**Comment by povoroznykmarta** [comment]
How do I sing out, I can't found it?

**Comment by hellachella** [comment]
![Uploading Screen Shot 2021-01-29 at 2.12.32 PM.png…]()

**Comment by hellachella** [comment]
![Screen Shot 2021-01-29 at 2 12 32 PM](https://user-images.githubusercontent.com/10465921/106317419-4c253a00-623c-11eb-82b0-d82e43837224.png)

**Comment by povoroznykmarta** [comment]
Okey, it worked and everything is good, but I can't submit it.

**Comment by hellachella** [comment]
What is it saying to you?

**Comment by povoroznykmarta** [comment]
<img width="838" alt="Screenshot 2021-01-30 at 11 11 10" src="https://user-images.githubusercontent.com/77293405/106352453-ec866780-62eb-11eb-9806-c8eed1cdd9eb.png">
